### PR TITLE
implement logrus wrapper

### PR DESCRIPTION
**Pull request by amery**

### CODE REVIEW

File: logrus/logrus.go [hunk 1]
```diff
@@ -72,27 +97,87 @@ func (rl *Logger) Fatal() slog.Logger {
...
+// New creates a slog.Logger adaptor using a logrus as backend
+func New(logger *logrus.Logger) slog.Logger {
+	if logger == nil {
+		return nil
+	}
+
+	return &Logger{
+		logger: logger,
+		entry:  logrus.NewEntry(logger),
+		level:  logrus.InfoLevel,
```
Comment (karasz) [inline logrus/logrus.go:167]: You are changing here the level of the received logger to Info. Why?

Comment (amery) [reply logrus/logrus.go:167]: I hadn't realised there was a received level 😅

Comment (karasz) [reply logrus/logrus.go:167]: logger.GetLevel()

Comment (amery) [reply logrus/logrus.go:167]: I thought that was linked to `IsLevelEnabled()`

Comment (amery) [reply logrus/logrus.go:167]: yes, according to https://pkg.go.dev/github.com/sirupsen/logrus#Logger.IsLevelEnabled the Level of the logger is the threshold to decide if a given level is enabled or not

Comment (karasz) [reply logrus/logrus.go:167]: Well.... in a sense the level of the Logger is the minimum level that will be logged so imho
```go
logger.GetLevel()
```
is what we need here and not the default 
```go 
InfoLevel
```

**Comment by amery** [comment]
rebased and changed to use logger.GetLevel() for the initial entry

**Review by karasz — APPROVED** [review]
LGTM

**Pull request merged**